### DISCUSSION TITLE
Accidentally did a typo and something happened. install git / install hit

### DISCUSSION THREAD

**Discussion by tomwebb2022** [Homebrew/discussions#3812] [Everyday usage]
So I'm learning via Odin Project and was tasked with installing Git. However, when I first tried I accidentally typed "brew install hit".

What is hit as something started to download?

**Comment by gromgit** [comment]
No idea:
```bash
$ brew info hit
Error: No available formula with the name "hit". Did you mean rhit, hdt, hut, bit, pit, vit, git or lit?
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.

$ brew info --cask hit
Error: Cask 'hit' is unavailable: No Cask with this name exists.
```
_ProTip_: When asking what happened when you did something, it helps to actually _show_ the messages you saw.

**Reply by tomwebb2022** [reply]
Not sure what has changed now but it doesn't seem to be coming up haha. I have no idea. Thanks though!
